System/Posix/Files.hsc

```diff
@@ -253,7 +253,7 @@ createSymbolicLink :: FilePath -> FilePath -> IO ()
 createSymbolicLink file1 file2 =
   withFilePath file1 $ \s1 ->
   withFilePath file2 $ \s2 ->
-  throwErrnoPathIfMinus1_ "createSymbolicLink" file2 (c_symlink s1 s2)
+  throwErrnoIfMinus1_ ("createSymbolicLink (target: "++file1++") (linkpath: "++file2++")") (c_symlink s1 s2)
```

Comment:
Should we be consistent and also use `name1` and `name2` here, just in the other functions?